### PR TITLE
Update of simple_forum_oc

### PR DESCRIPTION
**Pull request by Quent-in**

### CODE REVIEW

**Review by Gargron — APPROVED** [review]

File: config/locales/simple_form.oc.yml [hunk 1]
```diff
@@ -12,6 +12,8 @@ oc:
         data: Fichièr CSV exportat d’una autra instància Mastodon
       sessions:
         otp: Picatz lo còdi d’autentificacion en dos temps (Two factor code) de vòstre mobil o utilizatz un de vòstres còdis de recuperacion.
+  user:
```
Comment (ykzts) [inline config/locales/simple_form.oc.yml]: broken indent.

Comment (Gargron) [inline config/locales/simple_form.oc.yml]: Off the top of my head I don't even remember which level allowed_languages is supposed to go on. Should't it be just under labels.defaults?

**Review by Gargron — CHANGES_REQUESTED** [review]

File: config/locales/simple_form.oc.yml (hunk 1 above)
Comment (ykzts) [inline config/locales/simple_form.oc.yml]: This label is hint. The position is no problem. But indentation is broken.

Comment (Quent-in) [inline config/locales/simple_form.oc.yml]: Okay, what should I do?

Comment (ykzts) [inline config/locales/simple_form.oc.yml]: @Quent-in 5f9cb4888298528afc4866d5dc72c43877a41305 is the difference that makes similar changes in Japanese. please refer.

**Review by ykzts — APPROVED** [review]

**Review by Gargron — APPROVED** [review]

**Pull request merged**